main.js

```diff
@@ -0,0 +1,20 @@
+const showNav = document.getElementById('show-nav-btn');
+const hideNav = document.getElementById('hide-nav-btn');
+const mobileMenuLinks = document.getElementById('mobile-menu-links');
+const mobileMenu = document.getElementById('mobile-menu');
+
+showNav.addEventListener('click', () => {
+  if (mobileMenu.className === 'mobile-menu--hide') {
+    mobileMenu.className = 'mobile-menu--show';
+  }
```

Comment:
- [ ] When the hamburger button is clicked and menu appears the background should be blur, kindly check the design _PS: You can make background blur similar to this on CSS **filter: blur(8px);**_
![image](https://user-images.githubusercontent.com/62705947/215590349-09c11a93-e06e-44c4-afa2-9cc58bbc8c51.png)
![image](https://user-images.githubusercontent.com/62705947/215590404-06fde108-a5f0-4edf-82c3-1773abcfdf62.png)